src/DynamoCore/Graph/Workspaces/LayoutExtensions.cs

```diff
@@ -200,6 +213,29 @@ internal static List<GraphLayout.Graph> DoGraphAutoLayout(this WorkspaceModel wo
 
         }
 
+        /// <summary>
+        /// If a connector has connectorPins, their edges get added to the combined graph.
+        /// </summary>
+        /// <param name="combinedGraph"></param>
+        /// <param name="connector"></param>
+        private static void AddConnectorPinEdges(GraphLayout.Graph combinedGraph, ConnectorModel connector)
+        {
+            if (connector.ConnectorPinModels.Count < 1) return;
+
+            combinedGraph.AddEdge(connector.Start.Owner.GUID, connector.ConnectorPinModels[0].GUID,
+                connector.Start.Center.X, connector.Start.Center.Y, connector.ConnectorPinModels[0].CenterX, connector.ConnectorPinModels[0].CenterY);
```

Comment:
NIT: same comment as above, these gets a little hard to read

Reply:
Done.